interbtc-spec/docs/source/spec/staking.rst

```diff
@@ -192,6 +192,7 @@ Specification
 
 *Preconditions*
 
+* ``amount`` MUST NOT be zero
```

Comment:
I think we specify that a function fails if one of the preconditions does not hold. In this case, however, we return Ok. I think it's more of an optimization, right? I.e. if `amount` is zero, it just has no net effect?

Reply:
Do you think we should just remove it? It's certainly a precondition for the function to run, but you're correct in that there is no explicit error.

Reply:
If the behavior without the check would be a noop, then yes, I think we should remove this precondition in the spec